website/settings/defaults.py

```diff
@@ -1897,3 +1901,6 @@ class CeleryConfig:
 VERIFY_CHRONOS_SSL_CERT = not DEV_MODE
 # Maximum minutes we allow ChronosSubmission status to be stale (only update when user is requesting it)
 CHRONOS_SUBMISSION_UPDATE_TIME = timedelta(minutes=5)
+
+DS_METRICS_OSF_TOKEN = None
+DS_METRICS_BASE_FOLDER = None
```

Comment:
We want to be careful where we write these raw data files - lots of info in them.  We need to be sure they go to a specific place, and nowhere else.  Do we need any safeguards to prevent this? (Scenario where someone has data and playing around locally)

Reply:
If someone has production data, is actively using this script, and doesn't know to be careful with this, I'm not really sure what kind of safeguards to add, other than maybe a warning at the top of the management command, which I will add. If you have any other ideas, let me know.

Reply:
i know 😂 I just mean it's rare that we have a setup that's crossing boundaries between local development and outside things, but you're right, I'm just paranoid.

Reply:
Yeah, it's an odd one for sure.